src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeptionforservice.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User correct_user;
+    private static User correct_user2;
+    private static User null_password;
+    private static User null_login;
+    private static User incorrect_password;
+    private static User incorrect_login;
+    private static User incorrect_age;
+    private static User same_correct_user;
```

Comment:
Your Test should not have this many Users. It's enough to have only one user field. In each testing method you will assigned the proper value.